### PR TITLE
docker image tag

### PR DESCRIPTION
**Pull request by cccs-jc**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by cccs-rc — APPROVED** [review]
Sure, this looks like pipeline stuff

**Review by cccs-joel — APPROVED** [review]
That looks like our stuff.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -47,4 +47,5 @@ stages:
         tags: |
           $(BRANCH_NAME)
           $(BRANCH_NAME)_$(buildTimestamp)_b$(Build.BuildId)
+          latest
```
Comment (cccs-joel) [inline azure-pipelines.yml:50]: Ok, why do we need latest? Why don't we use master of this repo?

Comment (cccs-jc) [reply azure-pipelines.yml]: removed it

**Review by cccs-joel — APPROVED** [review]

**Pull request merged**